xcode/Subconscious/Shared/Components/Common/Profile/UserProfileView.swift

```diff
@@ -33,7 +33,7 @@ struct UserProfileView: View {
     var body: some View {
         let columnRecent = TabbedColumnItem(
```

Comment:
Nit: I tend to move any variables into private instance variables. It used to be the case that the view builder would get tripped up on the combination of statements and expressions.